pom.xml

```diff
@@ -937,7 +937,7 @@
               <systemId>http://maven.apache.org/xsd/decoration-1.0.0.xsd</systemId>
             </validationSet>
             <validationSet>
-              <dir>src/main/resources</dir>
+              <dir>src/test/resources</dir>
```

Comment:
Yes, we continue support this file in thitdparty artifacts that lead brute force algorithm to certain packages.

If they used our packages ( they did repacking of checkstyle with their custom check) ... we break this ... sad